src/record/record.js

```diff
@@ -255,9 +259,10 @@ Record.prototype.delete = function() {
 		return;
 	}
 	this.whenReady( function() {
-		this.emit( 'destroyPending' );
+		this.isReady = false;
```

Comment:
Setting isReady to false allows the record to be revived with pending operations.